src/layouts/Base.astro

```diff
@@ -64,6 +64,10 @@ const ogImage = createOgImage({
       name="description"
       content={description ? description : config.metadata.meta_description}
     />
+    <!-- Sitemap -->
+    <link rel="sitemap" href="/sitemap-index.xml" />
+    <!-- Manifest -->
+    <link rel="manifest" href="/site.webmanifest" />
```

Comment:
This line is a duplicate and is auto-generated.

Reply:
Done

Reply:
The manifest file (site.webmanifest) is called in the line # 70. I'ill revert this change and delete the sitemap line.

Reply:
Sorry the duplicate line has been deleted in another PR.